web/src/compose_tooltips.ts

```diff
@@ -161,18 +161,18 @@ export function initialize(): void {
             const narrow_filter = narrow_state.filter();
             let display_current_view;
             if (narrow_state.is_message_feed_visible()) {
-                if (narrow_filter === undefined) {
+                if (narrow_filter?.is_in_home()) {
```

Comment:
Could this just be an assert, given that we did `narrow_state.is_message_feed_visible()` just above? 

```
export function is_message_feed_visible(): boolean {                                                    
    // It's important that `message_lists.current` is the                                               
    // source of truth for this since during the initial app load,                                      
    // `message_lists.current` is `undefined` and we don't want                                         
    // to return `true` if we haven't loaded the message feed yet.                                      
    return message_lists.current !== undefined;                                                         
}                                                                                                       
```

Reply:
Yeah.